alibuild_helpers/__init__.py

```diff
@@ -1,4 +1,7 @@
 # Dummy file to package build_template.sh
-import utilities as utilities
-import analytics as analytics
-import log as log
+
+from __future__ import absolute_import
+
+from . import utilities
```

Comment:
Is this compatible with python on slc5? If not, can it be factored out in a separate PR?